.github/workflows/fornax-gh-pages.yml

```diff
@@ -0,0 +1,59 @@
+name: Deploy Docs
+
+on:
+  # Runs on pushes targeting the default branch
+  push:
+    branches: ["master"]
```

Comment:
@xperiandri we want PRs to the docs to also be tested before being merged, so please stop using the `branches` attribute, it is a bad practice